### ISSUE TITLE
How to request VATIC to not to split a video into multiple temporal-segments

### ISSUE THREAD

**Issue by ratnesh1729** [cvondrick/vatic#35]
For some reason I do not want video to split up (temporally) into multiple batches -> and then later merge annotations using Hungarian merge.
Therefore my query - How can I request Vatic to not to split up the video into multiple chunks ?
Thanks!

**Comment by Davidhw** [comment]
I agree that it would be useful to not need to split the videos. Splitting the videos introduces potential for error when merging the trajectories.

**Comment by aaman-ee** [comment]
This is helpful only when you have static objects that you want to detect or very smooth object trajectories throughout the video. 

For answering your question after the command

 $ turkic extract /path/to/video.mp4 /path/to/output/directory

go to the output directory and find the name of the last frame of the last segment (i.e. 1802.jpg).
Afterwards apply:

$ turkic load identifier /path/to/output/directory Label1 Label2 LabelN --length 1802 --offline

**Comment by Davidhw** [comment]
@aaman-ee Thank you! Why would the trajectories need to be especially smooth or static? If the video is loaded in only one clip, then the trajectories do not have to be merged together. As I understand it, you would simply have the manually created annotations without interpolation.

**Comment by EnnaSachdeva** [comment]
Any updates on if you could resolve this?
